src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,595 @@
+    global conn, cursor
+
+    iplist = []
+    domainlist = []
+    try:
+        logging.info("Starting insert into database...")
+
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+
+            logging.info(
+                "There was a connection made to"
+                " the database and the query was executed "
+            )
+
+            cursor = conn.cursor()
+
+            for ip in customerIP:
+                iplist.append(ip)
+
+                cursor.execute(
+                    f"insert into organizations(domain_name,"
+                    f" domain_ip,"
+                    f" date_saved) "
+                    f"values('{customer}',"
+                    f" '{ip}',"
+                    f"'{thedateToday}');"
+                )
+            for domain in customerRootDomain:
+                domainlist.append(domain)
+                cursor.execute(
+                    f"insert into domain_assets(domain_name,"
+                    f" domain_ip,"
+                    f" date_saved) "
+                    f"values('{customer}',"
+                    f" '{ip}', '{thedateToday}');"
+                )
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error("There was a problem logging into the psycopg database %s", err)
+    finally:
+        if conn is not None:
+            conn.commit()
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+    return iplist
+
+
+def getSubdomain(domain):
+    """Get all sub-domains from passed in root domain."""
+    allsubs = []
+
+    url = "https://domains-subdomains-discovery.whoisxmlapi.com/api/v1"
+    payload = json.dumps(
+        {
+            "apiKey": f"{API_WHOIS}",
+            "domains": {"include": [f"{domain}"]},
+            "subdomains": {"include": ["*"], "exclude": []},
+        }
+    )
+    headers = {"Content-Type": "application/json"}
+    response = requests.request("POST", url, headers=headers, data=payload)
+    data = response.json()
+
+    subdomains = data["domainsList"]
+    logging.info(subdomains)
+
+    subisolated = ""
+    for sub in subdomains:
+
+        if sub != f"www.{domain}":
+
+            logging.info(sub)
+            subisolated = sub.rsplit(".")[:-2]
+            logging.info(
+                "The whole sub is %s and the isolated sub is %s", sub, subisolated
+            )
+        allsubs.append(subisolated)
+
+    return subdomains, allsubs
+
+
+def theaddress(domain):
+    """Get actual IP address of domain."""
+    gettheAddress = ""
+    try:
+        gettheAddress = socket.gethostbyname(domain)
+    except socket.gaierror:
+        logging.info("There is a problem with the Domain that you selected")
+
+    return gettheAddress
+
+
+def getallsubdomainIPS(domain):
+    """Get a list if ip addresses associated with a subdomain."""
```

Comment:
Typo/capitalization:
```suggestion
    """Get a list of IP addresses associated with a subdomain."""
```